src/components/profile/user/profile/ProfileDetails.js

```diff
@@ -24,7 +24,7 @@ const ProfileDetails = ({ handle, bio = {}, createdAt }) => (
   <div className='profile-details'>
     <MuiLink
       component={Link}
-      to={`/user/${handle}`}
+      to={`/${handle}`}
```

Comment:
did you change it so that it does `website.com/myUserName`? that could be problematic if my username is too similar to a page name

Reply:
It's a temporarily fix, as I was having issues with the previously set address. I had intended on looking more into it after the main functionality was finished.

Reply:
```
const { pathname } = useLocation;

// ~ stuff ~

<MuiLink
  component={Link}
  to={`${pathname}/${handle}`}
/>

```

Reply:
Fixed